ui/app/AppLayouts/Chat/controls/community/PermissionsDropdown.qml

```diff
@@ -0,0 +1,202 @@
+        readonly property int extraMarginForText: 8
+
+        readonly property int separatorTopMargin: 4
+        readonly property int separatorBottomMargin: 12
+
+        readonly property int descriptionFontSize: 13
+        readonly property int descriptionLineHeight: 18
+
+        readonly property int buttonTopMargin: 32
+    }
+
+    ButtonGroup {
+        id: group
+    }
+
+    contentItem: ColumnLayout {
+        spacing: 0
+
+        Item {
+            Layout.fillWidth: true
+            Layout.preferredHeight: d.sectionHeight
+
+            StatusBaseText {
+                anchors.margins: d.extraMarginForText
+                anchors.top: parent.top
+                anchors.left: parent.left
+                anchors.right: parent.right
+                text: qsTr("Community")
+                color: Theme.palette.baseColor1
+                font.pixelSize: d.sectionFontSize
+                elide: Text.ElideRight
+            }
+        }
+
+        PermissionListItem {
+            readonly property int permissionType: PermissionTypes.Type.Admin
+            readonly property string description: {
+                const generalInfo = qsTr("Members who meet the requirements will be allowed to create and edit permissions, token sales, airdrops and subscriptions")
+                const warning = qsTr("Be careful with assigning this permission.")
+                const warningExplanation = qsTr("Only the community owner can modify admin permissions")
+
+                const warningStyled = `<font color="${Theme.palette.dangerColor1}">${warning}</font>`
+                return `${generalInfo}<br><br>${warningStyled} ${warningExplanation}`
+            }
+
+            title: qsTr("Become admin")
+            asset.name: "admin"
+            checked: d.initialPermissionType === permissionType
+            buttonGroup: group
+
+            Layout.fillWidth: true
+        }
+
+        PermissionListItem {
+            readonly property int permissionType: PermissionTypes.Type.Member
+            readonly property string description:
+                qsTr("Anyone who meets the requirements will be allowed to join your community")
+
+            title: qsTr("Become member")
+            asset.name: "in-contacts"
+            checked: d.initialPermissionType === permissionType
+            buttonGroup: group
+
+            Layout.fillWidth: true
+        }
+
+        Item {
+            Layout.fillWidth: true
+            Layout.preferredHeight: d.sectionHeight
+
+            StatusBaseText {
+                anchors.margins: d.extraMarginForText
+                anchors.top: parent.top
+                anchors.left: parent.left
+                anchors.right: parent.right
+                text: qsTr("Channels")
+                color: Theme.palette.baseColor1
+                font.pixelSize: d.sectionFontSize
+                elide: Text.ElideRight
+            }
+        }
+
+        PermissionListItem {
```

Comment:
what about having a Repeater here?

Reply:
I thought about that but finally I came to conclusion that while there are sections with custom text inside it would lead to more complicated and less readable code. In the future it would be shortened even more using inline components.